config/locales/views/annotation_categories/en.yml

```diff
@@ -4,7 +4,12 @@ en:
       success: "This annotation category was successfully created."
     destroy_annotation_text:
       confirm: "Deleting this annotation will remove the text from any files it may be attached to.  Are you sure you want to do this?"
+      error: 'Cannot delete this annotation'
```

Comment:
Perhaps give a more explicit reason (same for `update_annotation_text` below) Like "Cannot delete this annotation because it has a deductive value greater than 0 and has been applied to released results."

Reply:
Change key as well, deductive_annotation_released_error.